restservice/src/main/java/org/itevents/service/security/JWTCryptTokenService.java

```diff
@@ -40,7 +40,7 @@ public Token decrypt(String token) {
 
             return new Token(username, role);
         } catch (SignatureException | MalformedJwtException e) {
-            throw new CryptTokenServiceException("Don't trust the JWT");
+            throw new CryptTokenServiceException(e.getMessage(), e);
```

Comment:
i've had a lot of problems because of this message.
This message not explains what happened.